osu.Game.Tests/Visual/Multiplayer/TestSceneMultiSpectatorScreen.cs

```diff
@@ -349,15 +349,14 @@ public void TestPlayersLeaveWhileSpectating()
         }
 
         /// <summary>
-        /// Tests spectating with a gameplay start time set to a negative value.
-        /// Simulating beatmaps with high <see cref="BeatmapInfo.AudioLeadIn"/> or negative time storyboard elements.
+        /// Tests spectating with a beatmap that has a high lead-in value.
         /// </summary>
         [Test]
-        public void TestNegativeGameplayStartTime()
+        public void TestLeadIn()
```

Comment:
The modification of this test doesn't seem correct? Like yes maybe you did find a failing test, but isn't this losing coverage of the *other* case that was mentioned in this xmldoc previously (i.e. "negative time storyboard elements")? In other words, to me the old test case is testing something entirely different to what the new one is.

I would understand if a new test case was added next to this one, but I don't get replacing one with the other...

Reply:
It's generally a test case for adding a lead-in to a beatmap, and negative time storyboard elements are just another way to have lead-in, while `AudioLeadIn` is an explicit way of doing it. That's at least how I'm interpreting this.

Reply:
I would honestly prefer having both test cases in just for the peace of mind. Like the setup of the test has completely changed.

Reply:
Suuure I guess.

Reply:
Upon [follow-up discussion in discord](https://discord.com/channels/188630481301012481/188630652340404224/974406024821891143) I guess the old test scenario is already covered elsewhere where applicable, and doesn't really realistically apply to multi spectator. I'm going to hang the white flag on this, but generally would like to receive up-front explanations whenever test scenarios are replaced in this manner in the future.